lua/lean/infoview.lua

```diff
@@ -46,11 +41,7 @@ end
 -- either erase infoview information from table (erase=true)
 -- or indicate it has been closed (erase=false)
 local function close_win_raw(src_idx, erase)
-  if erase then
-    M._infoviews_open[src_idx] = nil
-  else
-    M._infoviews_open[src_idx] = false
-  end
+  M._infoviews_open[src_idx] = erase and nil or false
```

Comment:
It looks like this needs to be reverted as well (for the same reason mentioned earlier)

Reply:
Looking into this, it seems like Lua doesn't have an actual "ternary" operator - it can just be simulated using precedence and short-circuiting in specific cases.

Reply:
Thanks for fixing!